python-package/lightgbm/plotting.py

```diff
@@ -369,8 +402,15 @@ def plot_metric(booster, metric=None, dataset_names=None,
     return ax
 
 
-def _to_graphviz(tree_info, show_info, feature_names, precision=3,
-                 orientation='horizontal', constraints=None, **kwargs):
+def _to_graphviz(
+    tree_info: List[Dict[str, Any]], 
+    show_info: Optional[List[str]], 
+    feature_names: Optional[List[str]], 
+    precision: Optional[int] = 3,
+    orientation: Optional[str] = 'horizontal', 
+    constraints: Optional[List[int]]
```

Comment:
```suggestion
    constraints: Optional[List[int]],
```

This line is missing a comma (sorry, probably my mistake in my previous "suggest changes" comment)